wordpress-mobile/WordPress-iOS
# Convert CommentDetailViewController to a UIViewController

Ref: #17412 
Depends on: #17409 (to avoid massive merge conflicts)

This changes `CommentDetailViewController` from a `UITableViewController` to a `UIViewController`. 

This is to facilitate showing `ReplyTextView` and `SuggestionsTableView` when the `Reply` button is tapped. In a future PR, these views to be added to the new `containerStackView` when needed.

To test:
- Enable `newCommentDetail`.
- Go to My Site > Comments > Comment details.
- Verify the details appears and functions correctly.
- On a split view and non split view device, rotate and verify the view is displayed correctly.

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is flagged.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is flagged.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is flagged.

PR submission checklist:
